internal/action/init.go

```diff
@@ -203,7 +203,6 @@ func (s *Action) InitOnboarding(c *cli.Context) error {
 		return nil
 	}
 
-	ctx = out.AddPrefix(ctx, "[init] ")
```

Comment:
Not sure why you removed it, since it can be useful in the logs, but I guess we don't really need it neither.

Reply:
It was duplicated. In my tests there was still one `[init]` prefix left. Before this change there were two, which felt a bit weird.